src/commitizen/adapter.js

```diff
@@ -78,6 +78,7 @@ function getNearestNodeModulesDirectory(options) {
   let nodeModulesDirectories = findNodeModules(options);
   
    // Make sure we find a node_modules folder
+  /* istanbul ignore else */
```

Comment:
I thought ignoring the error cases was okay, testing them could prove challenging with the unit tests running on actual git commands instead of a mock layer.